docs/sdk/src/polkadot_sdk/polkadot.rs

```diff
@@ -47,10 +47,11 @@
 //!   Chain*").
 //! * (heterogenous) Sharded Execution: Yet, each parachain is free to have its own execution logic
 //!   (runtime), which also encompasses governance and sovereignty. Moreover, Polkadot ensures the
-//!   correct execution of all parachain, without having all of its validators re-execute all
-//!   parachain blocks. When seen from this perspective, the fact that Polkadot executes different
-//!   parachains means it is a platform that has fully delivered (the holy grail of) "Full Execution
-//!   Sharding". TODO: link to approval checking article. <https://github.com/paritytech/polkadot-sdk-docs/issues/66>
+//!   correct execution of all parachains, without having all of its validators re-execute all
+//!   parachain blocks. When seen from this perspective, the fact that different Polkadot validators
```

Comment:
```suggestion
//!   parachain blocks. When seen from this perspective, Polkadot achieves the ability to verify
//!   the validity of the block execution of multiple parachains using the same set of validators as
//!   the relay chain. In practice, this means that the shards (parachains) share the same economic
//!   security as the relay chain.
```

I think that something along these lines is less prone to confusion and more accurate.

Reply:
But capitalize Relay Chain.